### PR TITLE
DCA-220:Update products' base URL after restoring db

### PR DESCRIPTION
**Pull request by mmizin**
https://bulldog.internal.atlassian.com/browse/DCA-220

### CODE REVIEW

File: app/util/bitbucket/populate_db.sh [hunk 1]
```diff
@@ -94,7 +94,27 @@ else
...
-echo "Step2: Download DB dump"
+echo "Step2: Get DB Host"
+DB_HOST=$(sudo su -c "cat ${DB_CONFIG} | grep 'jdbc:postgresql' | cut -d'/' -f3 | cut -d':' -f1")
+if [[ -z ${DB_HOST} ]]; then
+  echo "DataBase URL was not found in ${DB_CONFIG}"
+  exit 1
+fi
+echo "DB_HOST=${DB_HOST}"
+
+echo "Step3: Write 'instance.url' property to file"
```
Comment (ometelytsia) [inline app/util/bitbucket/populate_db.sh:105]: Need to handle re-run case.
E.g. populate db step failed -> base_url exists and not empty, but the value in DB could be missing or incorrect.

Comment (mmizin) [reply app/util/bitbucket/populate_db.sh:105]: Add check if the file already exists and not empty

File: app/util/bitbucket/populate_db.sh [hunk 2]
```diff
@@ -94,7 +94,30 @@ else
...
+  echo "File ${BITBUCKET_BASE_URL_FILE} was found. Base url: $(cat ${BITBUCKET_BASE_URL_FILE})."
+else
+  PGPASSWORD=${BITBUCKET_DB_PASS} psql -h ${DB_HOST} -d ${BITBUCKET_DB_NAME} -U ${BITBUCKET_DB_USER} -c \
+  "select prop_value from app_property where prop_key='instance.url';" |
+  awk '/.htt/{print $1}' > ${BITBUCKET_BASE_URL_FILE}
+  if [[ ! -s ${BITBUCKET_BASE_URL_FILE} ]]; then
+    echo "Failed to get Base URL value form database. Check DB configuration variables."
+    exit 1
+  fi
+  echo "$(cat ${BITBUCKET_BASE_URL_FILE}) was written to the ${JIRA_BASE_URL_FILE} file."
```
Comment (SergeyMoroz0703) [inline app/util/bitbucket/populate_db.sh:117]: JIRA_BASE_URL_FILE in populate.db for bitbucket

Comment (mmizin) [reply app/util/bitbucket/populate_db.sh:117]: Yes, good catch! Thank you.

File: app/util/confluence/populate_db.sh [hunk 3]
```diff
@@ -99,42 +101,56 @@ else
...
+fi
+echo "DB_HOST=${DB_HOST}"
+
+echo "Step3: Write confluence baseUrl to file"
+CONFLUENCE_BASE_URL_FILE="base_url"
+if [[ -s ${CONFLUENCE_BASE_URL_FILE} ]];then
+  echo "File ${CONFLUENCE_BASE_URL_FILE} was found. Base url: $(cat ${CONFLUENCE_BASE_URL_FILE})."
+else
+  PGPASSWORD=${CONFLUENCE_DB_PASS} psql -h ${DB_HOST} -d ${CONFLUENCE_DB_NAME} -U ${CONFLUENCE_DB_USER} -c "${GET_CONFLUENCE_SETTINGS}" \
+  | awk -F' {2,}' '/.<baseUrl>/{print $2}' > ${CONFLUENCE_BASE_URL_FILE}
```
Comment (ometelytsia) [inline app/util/confluence/populate_db.sh:118]: ` | grep -i "<baseurl>"`

File: app/util/confluence/populate_db.sh [hunk 4]
```diff
@@ -16,20 +16,23 @@ CONFLUENCE_DB_NAME="confluence"
 CONFLUENCE_DB_USER="postgres"
 CONFLUENCE_DB_PASS="Password1!"
 
+# Confluence DB requests
+GET_CONFLUENCE_SETTINGS="select BANDANAVALUE from BANDANA where BANDANACONTEXT = '_GLOBAL' and BANDANAKEY = 'atlassian.confluence.settings';"
```
Comment (ometelytsia) [inline app/util/confluence/populate_db.sh:20]: let's rename to SELECT_CONFLUENCE_SETTING_SQL

File: app/util/confluence/populate_db.sh [hunk 5]
```diff
@@ -166,10 +182,39 @@ if [[ $? -ne 0 ]]; then
   exit 1
 fi
 
-echo "Step6: Start Confluence"
+echo "Step7: Update confluence baseUrl value in database"
+BASE_URL_TO_REPLACE=$(PGPASSWORD=${CONFLUENCE_DB_PASS} psql -h ${DB_HOST} -d ${CONFLUENCE_DB_NAME} -U ${CONFLUENCE_DB_USER} -c \
+"${GET_CONFLUENCE_SETTINGS}" | awk -F' {2,}' '/.<baseUrl>/{print $2}')
```
Comment (ometelytsia) [inline app/util/confluence/populate_db.sh:187]: `| grep -i "<baseurl>"`

**Comment by ometelytsia** [comment]
Please do not forget to remove update BaseURL step form jira, conf, bbs user guides.

**Pull request merged**